### PR TITLE
libGL: update to 19.1.3.

### PR DESCRIPTION
**Pull request by abenson**
Time for testing.

### CODE REVIEW

**Comment by abenson** [comment]
I'm comfortable with it.

**Comment by Johnnynator** [comment]
Can you look into https://github.com/void-linux/void-packages/issues/3293. Adding `llib32files="/usr/share/vulkan/icd.d/radeon_icd.x86.json"` and making sure that `library_path` in it points to `lib32` should be enough. (Same for Intel and nouveau probably)

**Comment by abenson** [comment]
Sure thing, that should be an easy enough fix.

**Comment by abenson** [comment]
That should work. It looks like the file only exists for `radeon` and `intel`.

File: srcpkgs/libGL/template [hunk 1]
```diff
@@ -96,6 +96,14 @@ post_install() {
 	vlicense docs/license.html
 	vmkdir usr/lib/xorg/modules/extensions
 	ln -s libglx-xorg.so ${DESTDIR}/usr/lib/xorg/modules/extensions/libglx.so
+	case $XBPS_TARGET_MACHINE in
+		i686*)
+			sed -i -e 's#/usr/lib/#/usr/lib32/#g' ${DESTDIR}/usr/share/vulkan/icd.d/*.json
+			;;
+		x86_64*)
```
Comment (Johnnynator) [inline srcpkgs/libGL/template]: That case is not needed, `/usr/lib` is always fine for x86_64

File: srcpkgs/libGL/template [hunk 2]
```diff
@@ -96,6 +96,14 @@ post_install() {
 	vlicense docs/license.html
 	vmkdir usr/lib/xorg/modules/extensions
 	ln -s libglx-xorg.so ${DESTDIR}/usr/lib/xorg/modules/extensions/libglx.so
+	case $XBPS_TARGET_MACHINE in
+		i686*)
+			sed -i -e 's#/usr/lib/#/usr/lib32/#g' ${DESTDIR}/usr/share/vulkan/icd.d/*.json
```
Comment (Johnnynator) [inline srcpkgs/libGL/template]: You could also use `vsed` to it throws an warning if it didn't change the file.

**Pull request merged**